### DISCUSSION TITLE
Error: Field is required: version - when upload New revision to grafana.com Dashboards

### DISCUSSION THREAD

**Discussion by masterwishx** [grafana/grafana#78995] [General]
when upload New revision to grafana.com Dashboards i have - Error: Field is required: version

![image](https://github.com/grafana/grafana/assets/28630321/e66fdcf8-fbf1-4802-ac8f-eec69c61be00)

version updated in json file :

![image](https://github.com/grafana/grafana/assets/28630321/7206618a-6876-4339-8a7b-b31f6571bb93)

![image](https://github.com/grafana/grafana/assets/28630321/653af621-3427-41ad-91ab-57728f8023ea)

**Comment by natellium** [comment]
Hey there! We want to let you all know that we are moving to Github Issues for feature requests. You can read more about the move https://github.com/grafana/grafana/discussions/73424.

This one in particular sounds like a potential bug, if you're still running into this issue please open a [bug report](https://github.com/grafana/grafana/issues/new?assignees=&labels=&projects=&template=0-bug-report.yaml&title=Product+Area%3A+Short+description+of+bug). Sorry for the inconvenience, and thanks for your contributions!
